gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -209,7 +209,7 @@ def queue_policy(queue_arn, topic_arn)
   "Id": "#{queue_arn}/SQSDefaultPolicy",
   "Statement": [
     {
-      "Sid": "#{Digest::MD5.hexdigest(queue_arn)}",
+      "Sid": "#{Rails.application.config.digest_class.hexdigest(queue_arn)}",
```

Comment:
>  For IAM policies, basic alphanumeric characters (A-Z,a-z,0-9) are the only allowed characters in the Sid value

so we should be good, assuming `Rails.application.config.digest_class.hexdigest` always returns hex stuff